libcaf_core/caf/flow/step/take_last.test.cpp

```diff
@@ -0,0 +1,88 @@
+// This file is part of CAF, the C++ Actor Framework. See the file LICENSE in
+// the main distribution directory for license terms and copyright or visit
+// https://github.com/actor-framework/actor-framework/blob/master/LICENSE.
+
+#include "caf/flow/step/take_last.hpp"
+
+#include "caf/test/caf_test_main.hpp"
+#include "caf/test/test.hpp"
+
+#include "caf/flow/scoped_coordinator.hpp"
+#include "caf/scheduled_actor/flow.hpp"
+
+#include <vector>
+
+using namespace caf;
+
+using caf::flow::make_observer;
+
+struct fixture {
+  flow::scoped_coordinator_ptr ctx = flow::make_scoped_coordinator();
+
+  template <class... Ts>
+  std::vector<int> ls(Ts... xs) {
+    return std::vector<int>{xs...};
```

Comment:
Can be shortened, never touches `this`:

```suggestion
  static auto ls(Ts... xs) {
    return std::vector<int>{xs...};
```